tests/scripts/thread-cert/Cert_9_2_02B_MGMTCommissionerSet.py

```diff
@@ -0,0 +1,157 @@
+        msg = commissioner_messages.next_coap_message('0.02', assert_enabled=False)
+        # Check coap URI and payload
+        assert msg.coap.uri_path == '/c/cs'
+        assert command.contains_tlvs(msg.coap.payload, [mesh_cop.SteeringData, mesh_cop.CommissionerSessionId])
+
+        # Check destination address
+        dest_addr = msg.ipv6_packet.ipv6_header.destination_address
+        rloc = ip_address(self.nodes[LEADER].get_addr_rloc())
+        aloc = ip_address(self.nodes[LEADER].get_addr_aloc())
+        assert dest_addr == rloc or dest_addr == aloc
+
+        # Step 5 - Leader sends MGMT_COMMISSIONER_SET.rsp to commissioner
+        leader_messages = self.simulator.get_messages_sent_by(LEADER)
+        msg = leader_messages.next_coap_message('2.04')
+        command.check_payload_same(msg.coap.payload, (mesh_cop.State(0x1),))
+
+        # Step 6 - Leader sends a multicast MLE Data Response
+        msg = leader_messages.next_mle_message(mle.CommandType.DATA_RESPONSE)
+        network_data = msg.assertMleMessageContainsTlv(NetworkData)
+        commissioning_data = command.get_sub_tlv(network_data.tlvs, CommissioningData)
+        assert commissioning_data is not None
+        assert commissioning_data.stable == 0
+        assert command.contains_tlvs(commissioning_data.sub_tlvs,
+                [mesh_cop.CommissionerSessionId, mesh_cop.SteeringData, mesh_cop.BorderAgentLocator])
+
+        # Step 9 - Harness instructs commissioner to send MGMT_COMMISSIONER_SET.req to Leader
+        border_agent_locator_tlv = mesh_cop.BorderAgentLocator(0x0400)
+        self.nodes[COMMISSIONER].commissioner_mgmtset_with_tlvs(
+            [steering_data_tlv, commissioner_session_id_tlv, border_agent_locator_tlv])
+        self.simulator.go(5)
+
+        # Step 10 - Leader responds to MGMT_COMMISSIONER_SET.req with MGMT_COMMISSIONER_SET.rsp
+        leader_messages = self.simulator.get_messages_sent_by(LEADER)
+        msg = leader_messages.next_coap_message('2.04')
+        command.check_payload_same(msg.coap.payload, (mesh_cop.State(0xFF),))
+
+        # Step 13 - Harness instructs commissioner to send MGMT_COMMISSIONER_SET.req to Leader
+        self.nodes[COMMISSIONER].commissioner_mgmtset_with_tlvs(
+            [commissioner_session_id_tlv, steering_data_tlv, mesh_cop.Channel(0x0, 0x0)])
+        self.simulator.go(5)
+
+        # Step 14 - Leader responds to MGMT_COMMISSIONER_SET.req with MGMT_COMMISSIONER_SET.rsp
+        leader_messages = self.simulator.get_messages_sent_by(LEADER)
+        msg = leader_messages.next_coap_message('2.04')
+        command.check_payload_same(msg.coap.payload, (mesh_cop.State(0x1),))
+
+        # Step 15 - Send ICMPv6 Echo Request to Commissioner
+        commissioner_rloc = self.nodes[COMMISSIONER].get_addr_rloc()
+        self.assertTrue(self.nodes[LEADER].ping(commissioner_rloc))
+
+    def skip_coap_messages(self, node_id, code):
```

Comment:
The code between line 81 and 85 of file tests/scripts/thread-cert/Cert_9_2_02A_MGMTCommissionerSet.py is the same with this function. Could we optimize it?